data/transactions/logic/opcodes_test.go

```diff
@@ -32,8 +32,57 @@ func TestOpSpecs(t *testing.T) {
 	}
 }
 
+func (os *OpSpec) equals(oso *OpSpec) bool {
+	if os.Opcode != oso.Opcode {
+		return false
+	}
+	if os.Name != oso.Name {
+		return false
+	}
+	if !reflect.DeepEqual(os.Args, oso.Args) {
+		return false
+	}
+	if !reflect.DeepEqual(os.Returns, oso.Returns) {
+		return false
+	}
+	if os.Version != oso.Version {
+		return false
+	}
+	if os.Modes != oso.Modes {
+		return false
+	}
+
+	return true
+}
+
+func TestOpcodesByVersionReordered(t *testing.T) {
+
+	// Make a copy to restore to the original
+	OpSpecsOrig := make([]OpSpec, len(OpSpecs))
+	for idx, opspec := range OpSpecs {
+		cp := opspec
+		OpSpecsOrig[idx] = cp
+	}
+	defer (func() {
```

Comment:
why the round brackets around the deferred function definition ?

Reply:
fixed.